lib/pbench/cli/agent/commands/tools/list.py

```diff
@@ -68,8 +73,13 @@ def execute(self) -> int:
                 for path in tg_dir:
                     host = path.name
                     tool_info[group][host] = {}
+                    label = path / "__label__"
+                    tool_info[group][host]["label"] = (
+                        None if not label.exists() else label.read_text().rstrip("\n")
+                    )
```

Comment:
I would have stated this "positively" (which makes the line shorter, among other things), and I would have used a local variable (to counteract Black's proclivities):
```python
                    v = label.read_text().rstrip("\n") if label.exists() else None
                    tool_info[group][host]["label"] = v
```